plugins/module_utils/network/sonic/config/bgp_neighbors_af/bgp_neighbors_af.py

```diff
@@ -220,6 +228,36 @@ def get_allowas_in(self, match, conf_neighbor_val, conf_afi, conf_safi):
                             mat_allowas_in = mat_nei_addr_fam.get('allowas_in', None)
         return mat_allowas_in
 
+    def get_ip_afi_cfg_payload(self, ip_afi):
```

Comment:
Since the logic of get_ip_afi_cfg_payload and get_prefix_limit_payload is identical in the bgp_neighbors and bgp_neighbors_af "config" files, it would be better to locate a single set of these two functions in the ".../bgp_utils.py" file and invoke the functions from each of the resource modules that use them. Use of the "ip_afi_cfg" local variable here is a better name than the one used for "bgp_neighbors", so the versions shown here would be good candidates for copying (moving) into the bgp_utils.py file.

Reply:
The requested change has been committed. This issue is resolved.